src/node_crypto.cc

```diff
@@ -990,7 +1001,7 @@ void SecureContext::LoadPKCS12(const FunctionCallbackInfo<Value>& args) {
   if (cert != nullptr)
     X509_free(cert);
   if (extra_certs != nullptr)
-    sk_X509_free(extra_certs);
+    sk_X509_pop_free(extra_certs, X509_free);
```

Comment:
This is unrelated bugfix for a memory leak, not sure if I should submit it separately. It won't have a test case anyway.